sdk/storage/azure-storage-queue/src/test/java/com/azure/storage/queue/spock/QueueServiceAPITests.groovy

```diff
@@ -0,0 +1,182 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.storage.queue.spock
+
+import com.azure.storage.queue.models.*
+import org.junit.Assert
+import spock.lang.Unroll
+
+class QueueServiceAPITests extends APISpec {
+
+    def setup() {
+        primaryQueueServiceClient = queueServiceBuilderHelper(interceptorManager).buildClient()
+    }
+    
+    def "Get queue client from queue service client"() {
```

Comment:
Added one more assertion for better code coverage

Reply:
I haven't found it yet, so correct me if I'm wrong, but there used to be a similar test as this one but it would make a service call to the queue it got to show that `getQueueClient`/`getQueueAsyncClient` doesn't create the queue in the service. If that test doesn't exist could we change this test to do that?

Reply:
I'm ok with having some validation that getQueueClient does not make a service request.

Reply:
The getClient did nothing but to initialize a new client class.
I don't think the service checking is needed as there is no logic within the service.
I have checked the class type.